logger/logger.go

```diff
@@ -9,23 +9,28 @@ import (
 
 func newLogger(c *Config) *Logger {
 	return &Logger{
-		Timestamp:   c.AdvancedSettings.TimeStampFormat,
-		Spacer:      c.AdvancedSettings.Spacer,
-		ErrorHeader: c.AdvancedSettings.Headers.Error,
-		InfoHeader:  c.AdvancedSettings.Headers.Info,
-		WarnHeader:  c.AdvancedSettings.Headers.Warn,
-		DebugHeader: c.AdvancedSettings.Headers.Debug,
+		Timestamp:         c.AdvancedSettings.TimeStampFormat,
+		Spacer:            c.AdvancedSettings.Spacer,
+		ErrorHeader:       c.AdvancedSettings.Headers.Error,
+		InfoHeader:        c.AdvancedSettings.Headers.Info,
+		WarnHeader:        c.AdvancedSettings.Headers.Warn,
+		DebugHeader:       c.AdvancedSettings.Headers.Debug,
+		ShowLogSystemName: *c.AdvancedSettings.ShowLogSystemName,
```

Comment:
This will be a nil pointer and cause a segfault if people don't have "showLogSystemName" in their config

Reply:
Whoopsies!